README.md

```diff
@@ -167,13 +167,114 @@ wrapping a component you intend to style, you'll need to make sure you accept
 the `className` as a prop and apply it to where you want the styles applied in
 your custom component (normally the root element).
 
-##### ...styles
```

Comment:
Yeah, I realized that and just moved this entire Examples section below SSR. I put it in the Terms and Concepts area, which was way off! 😅